src/Indigo/Contracts/Holdings/Types.hs

```diff
@@ -92,6 +92,27 @@ dummyMeta = TokenMeta [mt|KekToken|] [mt|Kek|] [mt|Ququareq|]
 
 type Storage = ML.StorageSkeleton StorageFields
 
+instance TypeHasDoc StorageFields where
+  typeDocName _ = "Holdings storage fields"
+  typeDocMdDescription =
+    "Additional contract fields that defines current contract state. \
+    \It stores meta-information about token. See `TokenMeta` type. \
+    \Information about the `owner` and the current `admin` of the contract. \
+    \Additionally they store optional address for currently used Safelist \
+    \contract, `paused`, `transferable` flags and `totalSupply` amount."
```

Comment:
```suggestion
    "Additional contract fields that define the current contract state. \
    \It stores meta-information about token (see `TokenMeta` type), \
    \information about the `owner` and the current `admin` of the contract. \
    \Additionally it stores an optional address for currently used Safelist \
    \contract, the `totalSupply` amount, and the `paused` and `transferable` flags."
```
Bit of rewording:
- the first `defines` I think should refer to the fields, even if we talk about them as a data type
- the third phrase has no verb, I think this can be solved by replacing the `.` with a `,` and merging it with the previous one.
- In the last one it now says "they", but it is no longer consistent with the singular "it" above, so that should be fixed
- I reordered the final list to make it more readable/clear